### ISSUE TITLE
Can not handle methods with 2D arrays inside

### ISSUE THREAD

**Issue by UlyssesWu** [CodeConnect/AliveFeedback#62]
Alive will show a lot of compile errors when trying to run following code.

```
    public static void Test()
    {
        string[,] array = new string[,]
        {
            {"cat", "dog"},
            {"bird", "fish"},
        };
        // ... Print out values.
        Console.WriteLine(array[0, 0]);
        Console.WriteLine(array[0, 1]);
        Console.WriteLine(array[1, 0]);
        Console.WriteLine(array[1, 1]);
    }
```

In fact I never use 2D arrays in my own code. But it's still an issue. I don't know if it only happens on my computer. I'm using VS2015 with Alive 1.2.0.

By the way, switch project's .NET Framework version when Alive is running may cause VS crash. That won't happen if manually stop Alive (by clicking the stop button) before switching.

**Comment by AmadeusW** [comment]
Thank you for the bug report. We'll get the array issue sorted out.

We're sorry for crashing your Visual Studio. Thank you for heads up and we will also address this issue.

**Comment by AmadeusW** [comment]
We've fixed the issue with the 2D arrays, but I still haven't investigated the crash. Expect an update with 2D array support later this week. Thanks for your feedback!

**Comment by AmadeusW** [comment]
Fixed in [1.2.1](https://github.com/CodeConnect/AliveFeedback/wiki/Changelog-1.2.1) :tropical_drink:
